### PR TITLE
SMD-191/add localstack

### PR DESCRIPTION
**Pull request by bobbi-hbrown**
https://dluhcdigital.atlassian.net/jira/software/c/projects/SMD/boards/140?modal=detail&selectedIssue=SMD-191

Sets up localstack on the docker runner, creates two S3 buckets and transfers the two excel test files into them for local development.

### How to test
The Localstack service should be available by building and running the compose repo. It is available at port 4566.

### CODE REVIEW

File: S3-files/example-total-grant-awarded.xlsx
Comment (cyrusdobbs) [inline S3-files/example-total-grant-awarded.xlsx:1]: Theses files looks fine for now. We'll want to make this particular one a fully mocked version of the real file later down the line so it's functional but that can be done in a separate ticket and PR.

**Review by cyrusdobbs — CHANGES_REQUESTED** [review]
Looks good

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .idea
 .env
+/volume
```
Comment (Connor-OS) [inline .gitignore:3]: Interested to know what is in this /volume directory? doesn't exist on my machine

**Review by Connor-OS — APPROVED** [review]
LGTM :)

**Review by cyrusdobbs — APPROVED** [review]

**Pull request merged**